src/test/regress/Makefile

```diff
@@ -10,6 +10,10 @@ ifndef MAJORVERSION
 MAJORVERSION := $(basename $(VERSION))
 endif
 
+ifeq ($(TESTHLL),true)
+HLLEXT := "--load-extension=hll"
```

Comment:
Ideally, the hll tests would just run as long as it is installed and we don't need to remember a flag.

What if we do something like this in the regression test?
```
DO $$
BEGIN
  CREATE EXTENSION hll;
EXCEPTION WHEN others THEN
  RAISE 'hll extension not installed';
END; 
$$;
```